tests/test_parser.py

```diff
@@ -23,7 +29,18 @@ def test_complex_outputs(sphinx_run, file_regression):
     # print(sphinx_run.status())
     assert sphinx_run.warnings() == ""
     assert sphinx_run.app.env.metadata == {
```

Comment:
as this is pretty long now, probably easier to add a `data_regression.check`

Reply:
same with the others

Reply:
ok this is a good opportunity to learn about `data_regression.check`. I guess that we will just have to update the regression templates any time that the jupytext version is updated

Reply:
Yeh, or if it's going to change to often, just do as a minimum `assert set(sphinx_run.app.env.metadata.keys()) == {"celltoolbar", ...}`